modules/olm-installing-opm.adoc

```diff
@@ -16,7 +16,9 @@ You can install the `opm` CLI tool on your Linux, macOS, or Windows workstation.
 
 .Prerequisites
 
-* `podman` version 1.4.4+
+* Linux dependencies:
```

Comment:
```suggestion
* For Linux, you must provide the following packages:
```
I'm ok with removing "workstation," but the prereq can't be a fragment.

Reply:
@kalexand-rh Exactly, your point is correct. I've modified it as a complete sentence.